runner/src/simulate.rs

```diff
@@ -65,6 +69,79 @@ impl Scenario {
     }
 }
 
+#[tracing::instrument]
+pub async fn simulate(opts: Opts) -> Result<()> {
+    let mut metrics = Metrics::init(&opts)?;
+
+    let f = File::open(opts.peers)?;
```

Comment:
It doesn't matter a ton here since it's early in the program, but prefer `tokio::fs` to `std::fs` for any file operations. If the fs calls end up blocking for any amount of time, it can cause issues with the tokio runtime.